app/src/main/res/layout/activity_main.xml

```diff
@@ -25,6 +26,7 @@
             android:id="@+id/edtKeyword"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:text="abc"
```

Comment:
WARNING: [HardcodedText] Hardcoded string "abc", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.